amostra/schemas/request.json

```diff
@@ -0,0 +1,35 @@
+{
+    "properties": {
```

Comment:
I think documents should have a set of standard fields including time and uid, the rest is simply in kwargs and up to the application to build the relationship